aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -160,21 +192,27 @@ module post_mint_reveal_nft::minting {
     /// The source certificate id not found in the signer's account.
     const ETOKEN_ID_NOT_FOUND: u64 = 15;
     /// Can only exchange after the reveal starts.
-    const ECANNOT_EXCHANGE_BEFORE_REVEAL_SRARTS: u64 = 16;
+    const ECANNOT_EXCHANGE_BEFORE_REVEAL_STARTS: u64 = 16;
     /// Can only add unique token uris.
     const EDUPLICATE_TOKEN_URI: u64 = 17;
+    /// The whitelist stage provided is not valid.
+    const EINVALID_STAGE: u64 = 18;
 
     /// Initialize NFTMintConfig for this module.
-    fun init_module(post_mint_reveal_nft_resource_account: &signer) {
-        let resource_signer_cap = resource_account::retrieve_resource_account_cap(post_mint_reveal_nft_resource_account, @source_addr);
-        move_to(post_mint_reveal_nft_resource_account, NFTMintConfig {
+    fun init_module(source: &signer) {
+        let seed_vec = bcs::to_bytes(&signer::address_of(source));
+        vector::append(&mut seed_vec, bcs::to_bytes(&timestamp::now_seconds()));
+        let (resource_account, resource_signer_cap) = account::create_resource_account(source, seed_vec);
```

Comment:
Why not just host this in the source account?

Reply:
yeah good point, updating now